### ISSUE TITLE
Remembering Custom Website passwords

### ISSUE THREAD

**Issue by whoave** [getferdi/ferdi#1118]
hello, ferdi is good but I a added 3 Custom Website. It logouts from the websites constantly , thats okay but writing passwords is a problem, how can I make ferdi remember passwords?

**Comment by whoave** [comment]
Any news?

**Comment by niutech** [comment]
You can use `user.js` for that, setting e.g. `document.getElementById('password').value = '...';`

**Comment by vraravam** [comment]
@niutech : Though your suggestion is a novel idea, I would not recommend it. This is what password managers are for.

**Comment by niutech** [comment]
@vraravam Yes, it's just a workaround. You can make a proper password manager using [keytar](https://atom.github.io/node-keytar/) ([howto](https://cameronnokes.com/blog/how-to-securely-store-sensitive-information-in-electron-with-node-keytar/)) or see how [Neutron did it](https://github.com/webcatalog/neutron/pull/609)
